config/crd/bases/anywhere.eks.amazonaws.com_bundles.yaml

```diff
@@ -216,7 +216,7 @@ spec:
                       - metadata
                       - version
                       type: object
-                    bottlerocketAdmin:
+                    bottlerocket:
```

Comment:
If we upgrade an older cluster to a newer one with these changes and the older bundle is referenced, would this be a problem? Do we need to phase it out instead?

Reply:
I don't think it would be an issue: we have the core components - eksa components upgrade in the upgrade workflow. This will update the eksa CRD (including the bundle CRD) and apply the new bundle spec, before the actual cluster upgrade.